cornichon-core/src/main/scala/com/github/agourlay/cornichon/dsl/Dsl.scala

```diff
@@ -23,11 +23,23 @@ import scala.concurrent.duration.FiniteDuration
 trait Dsl extends ProvidedInstances {
   this: BaseFeature ⇒
 
-  def Feature(name: String, ignored: Boolean = false) =
-    BodyElementCollector[ScenarioDef, FeatureDef](scenarios ⇒ FeatureDef(name, scenarios, ignored))
+  def Feature(name: String) = FeatureBuilder(name)
 
-  def Scenario(name: String, ignored: Boolean = false) =
-    BodyElementCollector[Step, ScenarioDef](steps ⇒ ScenarioDef(name, steps, ignored))
+  case class FeatureBuilder(name: String, _ignoredBecause: Option[String] = None) {
+    def ignoredBecause(reason: String) = copy(_ignoredBecause = Some(reason))
+  }
+
+  implicit def featureBuilder(f: FeatureBuilder): BodyElementCollector[ScenarioDef, FeatureDef] =
```

Comment:
I have not thought about the integration with the macro, nicely done 👍 

I guess those dedicated builders might be useful in the future to add more knobs on the ```Scenario``` and ```Feature```.